src/SFA.DAS.Commitments.Infrastructure.UnitTests/Services/ApprenticeshipEventsTests/ApprenticeshipEventsTestsBase.cs

```diff
@@ -35,7 +35,7 @@ public void Arrange()
                 EmployerAccountId = 987,
                 LegalEntityId = "LE ID",
                 LegalEntityName = "LE Name",
-                LegalEntityOrganisationType = OrganisationType.CompaniesHouse
+                LegalEntityOrganisationType = SFA.DAS.Common.Domain.Types.OrganisationType.CompaniesHouse
```

Comment:
This is explicitly named here to avoid accidental and contamination with the still available, but now obsolete, enum of the same name in the Commitments namespace